### ISSUE TITLE
SelectBooleanCheckbox: Hidden elements should have the aria-hidden attribute

### ISSUE THREAD

**Issue by trajano** [primefaces/primefaces#2720]
## 1) Environment
- PrimeFaces version: 6.1
- Application server + version: WebSphere 8.5.5.12
- Affected browsers: Internet Explorer/ Chrome/Firefox

## 2) Expected behavior

```
<p:selectBooleanCheckbox
  value="#{applicationBean.application.firstNation}"
  itemLabel="First Nation" />
```

Should render 

```
<div id="applicationForm:j_id1447386971_3_3b2db25b" class="ui-chkbox ui-widget"<div class="ui-helper-hidden-accessible">
  <input id="applicationForm:j_id1447386971_3_3b2db25b_input" name="applicationForm:j_id1447386971_3_3b2db25b_input" autocomplete="off" aria-checked="false" type="checkbox" hidden>
...
```

## 3) Actual behavior

```
<div id="applicationForm:j_id1447386971_3_3b2db25b" class="ui-chkbox ui-widget"<div class="ui-helper-hidden-accessible">
  <input id="applicationForm:j_id1447386971_3_3b2db25b_input" name="applicationForm:j_id1447386971_3_3b2db25b_input" autocomplete="off" aria-checked="false" type="checkbox" >
...
```

Missing `hidden` element causing accessibility tool to flag it.

## 4) Steps to reproduce

```
<p:selectBooleanCheckbox
  value="#{applicationBean.application.firstNation}"
  itemLabel="First Nation" />
```

## 5) Sample XHTML

```
<p:selectBooleanCheckbox
  value="#{applicationBean.application.firstNation}"
  itemLabel="First Nation" />
```

## 6) Sample bean

```
    Boolean firstNation;
    public Boolean getFirstNation() {
        return firstNation;
    }
    public void setFirstNation(final Boolean a) {
       firstNation = a;
    }
```

**Comment by trajano** [comment]
In

https://github.com/primefaces/primefaces/blob/master/src/main/java/org/primefaces/component/selectbooleancheckbox/SelectBooleanCheckboxRenderer.java#L102

Something like 
```
writer.writeAttribute("hidden",null, null);
```
should be added

**Comment by tandraschko** [comment]
because?

**Comment by trajano** [comment]
As I specified:

> Missing hidden element causing accessibility tool to flag it.

**Comment by Rapster** [comment]
Feel free to provide a PR then ;-)

**Comment by trajano** [comment]
Just need to find time to set up and do it

**Comment by melloware** [comment]
I submitted PR https://github.com/primefaces/primefaces/pull/2743

I found everywhere "ui-helper-hidden-accessible" was being used to hide an input and I added the HTML hidden attribute to the input or select component.   Please review!

**Comment by tandraschko** [comment]
Thanks for the pR!

**Comment by tandraschko** [comment]
Reverted as it breaks #2807

**Comment by ssibitz** [comment]
Now i see what the customer with #2720 means:

Not the "hidden" - tag
but the tag for **screenreader's** should be hidden by setting this attribute:

`aria-hidden="true"`

I think that's a mistake from the customer...

**Comment by melloware** [comment]
yep you are right I will fix

**Comment by ssibitz** [comment]
This aria-hidden BugFix will fix both issues: #2720 and #2807 :-)
